src/app/signup/page.jsx

```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+import styles from "../stylesheet/signup.module.css";
+import SocialLoging from "../components/SocialLogin";
+import TextInput from "../components/TextInput";
+import Label from "../components/Label";
+
+export default function signUp() {
+  return (
+    <main className={styles.main}>
+      <div className={styles.container}>
+        <Image
+          className={styles.logo}
+          src="/Frame.svg"
+          alt="Ember logo"
+          width={135}
+          height={54}
+          priority
+        />
+        <h2>Sign up for an account.</h2>
+        <p className={styles.subtitle}>Send, spend and save smarter</p>
+        <SocialLoging />
+        <div className={styles.login_email}>
+          <div className={styles.login_email__line}>
+            <hr />
+          </div>
+
+          <span className={styles.login_email__text}>or with e-main</span>
+
+          <div className={styles.login_email__line}>
+            <hr />
+          </div>
+        </div>
+        <form action="" className={styles.form}>
+          <div className={styles.inputs__responsive}>
+            <div className={styles.input__container}>
+              <Label id="first_name" class="sr-only" text="First name" />
+              <TextInput
+                type="text"
+                id="first_name"
+                name="first_name"
+                placeholder="First name"
+              />
+            </div>
+            <div className={styles.input__container}>
+              <Label id="last_name" class="sr-only" text="Last name" />
+              <TextInput
+                type="text"
+                id="last_name"
+                name="last_name"
+                placeholder="Last name"
+              />
+            </div>
+          </div>
+          <div className={styles.input__container}>
+            <Label id="email" class="sr-only" text="Email" />
+            <TextInput
+              type="text"
+              id="email"
+              name="email"
+              placeholder="Email"
+            />
+          </div>
+          <div className={styles.input__container}>
+            <Label id="password" class="sr-only" text="Password" />
+            <TextInput
+              type="password"
+              id="password"
+              name="password"
+              placeholder="Password"
+            />
+          </div>
+          <div className={`${styles.terms__container} ${styles['terms__container--responsive']}`}>
+            <p className={styles.terms}>
+              <span className={styles.terms__span}>
+                By creating an account, you agreeing to our
+              </span>{" "}
```

Comment:
What is this `{" "}` doing here? If the space character is not enough for you, you can use the spacing character entity `&nbsp;`

https://www.freecodecamp.org/news/html-space-how-to-add-a-non-breaking-space-with-the-nbsp-character-entity/